Test fails when run individually: src/vs/workbench/contrib/terminal/test/browser/xterm/xtermTerminal.test.ts

After much installing and compiling, when I run 
```
$ yarn test-browser
...
XtermTerminal - /CHROMIUM/
  theme - /CHROMIUM/
    ✓ should react to and apply theme updates
...
8121 passing
```

But when I run the test individually, it fails:
```
yarn test-browser --browser chromium --run src/vs/workbench/contrib/terminal/test/browser/xterm/xtermTerminal.test.ts
yarn run v1.22.19
$ npx playwright install && node test/unit/browser/index.js --browser chromium --run src/vs/workbench/contrib/terminal/test/browser/xterm/xtermTerminal.test.ts


  XtermTerminal - /CHROMIUM/
    ✔ should use fallback dimensions of 80x30
    theme - /CHROMIUM/
      ✔ should apply correct background color based on getBackgroundColor
      1) should react to and apply theme changes
    renderers - /CHROMIUM/
      - should re-evaluate gpu acceleration auto when the setting is changed


  2 passing (44ms)
  1 pending
  1 failing

  1) XtermTerminal
       theme
         should react to and apply theme changes:

     
      + expected - actual

       {
         "background": [undefined]
      -  "black": [undefined]
      -  "blue": [undefined]
      -  "brightBlack": [undefined]
      -  "brightBlue": [undefined]
      -  "brightCyan": [undefined]
      -  "brightGreen": [undefined]
      -  "brightMagenta": [undefined]
      -  "brightRed": [undefined]
      -  "brightWhite": [undefined]
      -  "brightYellow": [undefined]
      +  "black": "#010000"
      +  "blue": "#050000"
      +  "brightBlack": "#090000"
      +  "brightBlue": "#130000"
      +  "brightCyan": "#150000"
      +  "brightGreen": "#110000"
      +  "brightMagenta": "#140000"
      +  "brightRed": "#100000"
      +  "brightWhite": "#160000"
      +  "brightYellow": "#120000"
         "cursor": "#000300"
         "cursorAccent": "#000400"
      -  "cyan": [undefined]
      +  "cyan": "#070000"
         "foreground": "#000200"
      -  "green": [undefined]
      -  "magenta": [undefined]
      -  "red": [undefined]
      +  "green": "#030000"
      +  "magenta": "#060000"
      +  "red": "#020000"
         "selectionBackground": "#000500"
         "selectionForeground": [undefined]
         "selectionInactiveBackground": "#000600"
      -  "white": [undefined]
      -  "yellow": [undefined]
      +  "white": "#080000"
      +  "yellow": "#040000"
       }
     
  AssertionError: {"background":"undefined","foreground":"#000200","cursor":"#000300","cursorAccent":"#000400","selectionBackground":"#000500","se deepStrictEqual {"background":"undefined","foreground":"#000200","cursor":"#000300","cursorAccent":"#000400","selectionBackground":"#000500","se
      at Context.<anonymous> (http://localhost:40779/9b4192d4d6856fd078fe70a5e0405047/out/vs/workbench/contrib/terminal/test/browser/xterm/xtermTerminal.test.js:148:46)
````

Looking at src/vs/workbench/contrib/terminal/test/browser/terminalService.test.ts, the issue seems to be that the test does not call `registerColors()`:
https://github.com/microsoft/vscode/blob/f1b07bd25dfad64b0167beb15359ae573aecd2cc/src/vs/workbench/contrib/terminal/common/terminalColorRegistry.ts#L271-L277

anywhere, which initializes `ansiColorIdentifiers`, yet then accesses it here: https://github.com/microsoft/vscode/blob/f1b07bd25dfad64b0167beb15359ae573aecd2cc/src/vs/workbench/contrib/terminal/browser/xterm/xtermTerminal.ts#L912

When run with other tests, they `registerColors` which is leaked via global scope. 

I will try to send a PR to add that call.

--- a/src/vs/workbench/contrib/terminal/test/browser/xterm/xtermTerminal.test.ts
+++ b/src/vs/workbench/contrib/terminal/test/browser/xterm/xtermTerminal.test.ts
@@ -16,7 +16,7 @@ import { IThemeService } from 'vs/platform/theme/common/themeService';
 import { IViewDescriptor, IViewDescriptorService, ViewContainerLocation } from 'vs/workbench/common/views';
 import { IEditorOptions } from 'vs/editor/common/config/editorOptions';
 import { Emitter } from 'vs/base/common/event';
-import { TERMINAL_BACKGROUND_COLOR, TERMINAL_FOREGROUND_COLOR, TERMINAL_CURSOR_FOREGROUND_COLOR, TERMINAL_CURSOR_BACKGROUND_COLOR, TERMINAL_SELECTION_BACKGROUND_COLOR, TERMINAL_SELECTION_FOREGROUND_COLOR, TERMINAL_INACTIVE_SELECTION_BACKGROUND_COLOR } from 'vs/workbench/contrib/terminal/common/terminalColorRegistry';
+import { registerColors, TERMINAL_BACKGROUND_COLOR, TERMINAL_FOREGROUND_COLOR, TERMINAL_CURSOR_FOREGROUND_COLOR, TERMINAL_CURSOR_BACKGROUND_COLOR, TERMINAL_SELECTION_BACKGROUND_COLOR, TERMINAL_SELECTION_FOREGROUND_COLOR, TERMINAL_INACTIVE_SELECTION_BACKGROUND_COLOR } from 'vs/workbench/contrib/terminal/common/terminalColorRegistry';
 import { PANEL_BACKGROUND, SIDE_BAR_BACKGROUND } from 'vs/workbench/common/theme';
 import type { WebglAddon } from '@xterm/addon-webgl';
 import { NullLogService } from 'vs/platform/log/common/log';
@@ -35,6 +35,8 @@ import { IContextKeyService } from 'vs/platform/contextkey/common/contextkey';
 import { ITerminalLogService } from 'vs/platform/terminal/common/terminal';
 import { ensureNoDisposablesAreLeakedInTestSuite } from 'vs/base/test/common/utils';
 import { ILayoutService } from 'vs/platform/layout/browser/layoutService';
+
+registerColors();
 
 class TestWebglAddon implements WebglAddon {
 	static shouldThrow = false;

